ts/RailroadMap.ts

```diff
@@ -1466,14 +1466,19 @@ export class RailroadMap {
             case 'ballast_h01':
             case 'ballast_h05':
             case 'ballast_h10':
+            case 'ballast_h01_snow':
+            case 'ballast_h05_snow':
+            case 'ballast_h10_snow':
                 makePath(this.layers.groundworks, ['grade', spline.type.substring(spline.type.length - 3)]);
```

Comment:
It would be good to create a new style for the snow splines, since they aren't brown.

Reply:
well it would be but at the moment seems people just want to see them